hadoop-ozone/dist/src/main/compose/ozone-topology/docker-compose.yaml

```diff
@@ -147,6 +147,22 @@ services:
       networks:
          net:
             ipv4_address: 10.5.0.71
+   recon:
+     image: ${OZONE_RUNNER_IMAGE}:${OZONE_RUNNER_VERSION}
+     privileged: true #required by the profiler
+     volumes:
+       - ../..:/opt/hadoop
+     ports:
+       - 9888:9888
+     env_file:
+       - docker-config
```

Comment:
nit
```suggestion
       - ./docker-config
```

Reply:
@smengcl are you suggesting this for consistency with the rest of the services in this file?

Reply:
Ah, yes. Though it is [not strictly necessary](https://docs.docker.com/compose/environment-variables/#the-env_file-configuration-option) (unlike `volume` when we want to map a file from current working directory)

Reply:
Thanks for noticing this, I updated my patch with this change.